### PR TITLE
The space confige file name is wrong

### PR DESCRIPTION
**Pull request by roshi-bushido**

### CODE REVIEW

File: anypoint-platform-on-premises/v/1.5.0/configuring-anypoint-platform-for-pcf.adoc [hunk 1]
```diff
@@ -59,21 +59,21 @@ Each instance of the Space Configuration Tool is linked to one PCF space. Theref
...
 
 . Download the Space Configuration Tool here link:https://s3-us-west-1.amazonaws.com/anypoint-anywhere/1.5.0-LA/PCF+Installers/space_configuration_tool_1.5.0-beta.zip[zip file]
-. Open the `cloud_settings.yml` file and edit the properties in it to match your account, following the reference in <<Cloud Settings File, the section below>>.
+. Open the `space_settings.yml` file and edit the properties in it to match your account, following the reference in <<Cloud Settings File, the section below>>.
 . Run the script in your Cloud Foundry environment through the following command:
 
 +
 ----
-python cloud_installer.py -i cloud_settings.yml
+python cloud_installer.py -i space_settings.yml
```
Comment (harirongali) [inline anypoint-platform-on-premises/v/1.5.0/configuring-anypoint-platform-for-pcf.adoc]: python space_configuration_tool.py -i space_settings.yml

**Comment by mohanpindyala** [comment]
LGTM

File: anypoint-platform-on-premises/v/1.5.0/configuring-anypoint-platform-for-pcf.adoc [hunk 2]
```diff
@@ -59,21 +59,21 @@ Each instance of the Space Configuration Tool is linked to one PCF space. Theref
 To install the Space Configuration Tool, follow the steps below:
 
 . Download the Space Configuration Tool here link:https://s3-us-west-1.amazonaws.com/anypoint-anywhere/1.5.0-LA/PCF+Installers/space_configuration_tool_1.5.0-beta.zip[zip file]
-. Open the `cloud_settings.yml` file and edit the properties in it to match your account, following the reference in <<Cloud Settings File, the section below>>.
+. Open the `space_settings.yml` file and edit the properties in it to match your account, following the reference in <<Cloud Settings File, the section below>>.
```
Comment (harirongali) [inline anypoint-platform-on-premises/v/1.5.0/configuring-anypoint-platform-for-pcf.adoc]: <<Cloud Settings File, the section below>>  ==> <<Space Settings File, the section below>>

**Comment by sleslie23** [comment]
@roshi-bushido Thanks for catching this, Matias!

**Pull request merged**